mycroft/skills/phillips_hue/test/intent/sample3.intent.json

```diff
@@ -0,0 +1,7 @@
+{
+  "utterance": "turn down the lights",
+  "intent_type": "DecreaseBrightnessIntent",
+  "intent": {
+    "DecreaseBrightnessKeyword": "turn down the lights"
+  }
+}
```

Comment:
Please, add two intent tests for each intent to provide more coverage for your vocab.